engine/src/opensslsocket.cpp

```diff
@@ -965,10 +970,8 @@ void MCS_write_socket(const MCStringRef d, MCSocket *s, MCObject *optr, MCNameRe
 	{
 		// MW-2012-11-13: [[ Bug 10516 ]] Set the 'broadcast' flag based on whether the
 		//   user has enabled broadcast addresses.
-		int t_broadcast;
-		t_broadcast = MCallowdatagrambroadcasts ? 1 : 0;
-		setsockopt(s -> fd, SOL_SOCKET, SO_BROADCAST, (const char *)&t_broadcast, sizeof(t_broadcast));
-	
+		// HH-2017-01-26: [[ Bug 18454 ]] Move socket option setting to MCS_open_socket
+
```

Comment:
The comments here (on lines 971, 972 and 973) can be removed. Since git provides fully traceable source history, they don't serve much purpose.